packages/airnode-starter/README.md

```diff
@@ -0,0 +1,193 @@
+
+Run the following script and provide this information:
+```sh
+yarn choose-integration
+```
+
+### 2. (Only if using local blockchain) Start hardhat network
+
+Run:
+```sh
+yarn eth-node
+```
+
+which will spin up a [hardhat network](https://hardhat.org/hardhat-network/) on your machine.
+
+### 3. Deploy the RRP contract
+
+The RRP is a contract that through with the [requester](link) triggers a request for Airnode. This contract is common for all Airnodes and requesters on a chain.
+
+API3 team will deploy this contract for the most popular chains at some point, but for now you have to deploy it yourself using:
+
+```sh
+yarn deploy-rrp
+```
+
+### 4. (Only if deploying to AWS) Create AWS secrets file
+
+If you intend to deploy Airnode on AWS, you will need to specify the credentials which will be used by the [airnode
+deployer](link). If you are not sure where to find these or how to create an AWS account, follow [this
+video](https://www.youtube.com/watch?v=KngM5bfpttA).
+
+After you know the secrets, run the following script to specify them.
+
+```sh
+yarn create-aws-secrets
+```
+
+The command will generate an `aws.env` file with the entered secrets. This file be used only by the deployer and will
+never leave your machine.
+
+### 5. Create Airnode secrets
+
+Airnode is configured by two files - `config.json` and `secrets.env`. Run the script below to create the `secrets.env` file with the necessary values.
+
+```sh
+yarn create-airnode-secrets
+```
+
+_Refer to the [documentation](https://docs.api3.org/airnode/next/grp-providers/guides/build-an-airnode/configuring-airnode.html) for more details_
+
+### 6. Build docker artifacts
+
+Our docker images are based on a common container which we call "artifacts". This intermediate container is then used by
+both [deployer](https://github.com/api3dao/airnode/tree/master/packages/deployer) and
+[airnode](https://github.com/api3dao/airnode/tree/master/packages/node). You can build the artifacts container by running:
+
+```sh
+yarn rebuild-artifacts-container
+```
+
+### 7. (Only if deploying to AWS) Build deployer container
+
+```sh
+yarn rebuild-deployer-container
+```
+
+This command will facilitate the previously built artifacts container to build the deployer.
+
+### 8. (Only if deploying to AWS) Deploy Airnode
+
+Now you're ready to deploy Airnode on AWS. Just run:
+
+```sh
+yarn deploy-airnode
+```
+
+This command will just use the [airnode deployer](https://github.com/api3dao/airnode/tree/master/packages/deployer) package. Deployment may take some time so be patient.
+
+### 9. (Only if running Airnode locally) Build Airnode docker container
+
+```sh
+yarn rebuild-airnode-container
+```
+
+This command will facilitate the previously built artifacts container to build the containerized version of Airnode
+which you can run locally.
+
+### 10. (Only if running Airnode locally) Run the Airnode container
+
+```sh
+yarn run-airnode-locally
```

Comment:
Despite succeeding with Step 9 and having hardhat node (still) running in a separate terminal, I received (every minute):
```bash
> yarn run-airnode-locally
yarn run v1.22.11
$ ts-node scripts/run-airnode-locally.ts
[2021-09-29 07:14:05.735] INFO Coordinator starting...                                                           Coordinator-ID:36a29d6acfe41733
[2021-09-29 07:14:08.074] ERROR Failed to initialize provider:exampleProvider                                     Coordinator-ID:36a29d6acfe41733
[2021-09-29 07:14:08.075] ERROR Error: could not detect network (event="noNetwork", code=NETWORK_ERROR, version=providers/5.4.4)
    at Logger.makeError (/app/node_modules/@ethersproject/logger/lib/index.js:187:21)
    at Logger.throwError (/app/node_modules/@ethersproject/logger/lib/index.js:196:20)
    at StaticJsonRpcProvider.<anonymous> (/app/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:491:54)
    at step (/app/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
    at Object.throw (/app/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
    at rejected (/app/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)  Coordinator-ID:36a29d6acfe41733
[2021-09-29 07:14:08.075] INFO Forking to initialize providers complete                                          Coordinator-ID:36a29d6acfe41733
[2021-09-29 07:14:08.076] INFO No actionable requests detected. Returning...                                     Coordinator-ID:36a29d6acfe41733
[2021-09-29 07:14:08.076] INFO Coordinator completed at 2021-09-29 07:14:08. Total time: 2703ms                  Coordinator-ID:36a29d6acfe41733
```

Reply:
Are you using windows?

Reply:
My OS is Windows but I'm using WSL2 so it could be something strange about docker container communication.

Reply:
Yeah, if you use WSL, than it should all go smooth... Can you try once again, just in case?

Reply:
I rebuilt everything and reran from the beginning. Same error unfortunately. To ensure it wasn't a user error on my part, I spun up a linux EC2 instance, walked through the tutorial, and this step now worked correctly (although I then failed at  `api3-admin: Permission denied` as mentioned in the other comment).

Reply:
Yeah, you're right. I've reproduced this on WSL2 as well.

The problem is caused by how the networks are managed when using docker and WSL:
- https://www.beyondjava.net/docker-wsl-network

There are also more issues, but I wasn't able to fix this just yet... I've created https://api3dao.atlassian.net/browse/AN-251 as I expect to take this more time (and knowledge).

In the meantime I'll mention that this flow (airnode container running locally) is not working on WSL2 yet...

Reply:
👍 thanks for the investigation and confirming. It makes sense to not have this blocking and instead mention it's not working yet.

Reply:
Luckily, I found a fix for this. But I'll do it in a separate PR.